nodes/Compare.py

```diff
@@ -0,0 +1,2 @@
+if (3 > 2):
```

Comment:
Oh, but you could also just do `3 > 2` as a single expression, rather than the `if`, I assume.